pyproject.toml

```diff
@@ -1,11 +1,46 @@
-[build-system]
+[project]
 name = "ElectionLeaflets"
-version = "3.0-alpha"
-requires = [
-    "setuptools >= 35.0.2",
-    "setuptools_scm >= 2.0.0, <3"
+version = "0.1.0"
```

Comment:
In our pyproject.toml we should set

```toml
[tool.uv]
package = false
```

This tells UV this is an application not a package and will prevent us from pushing it to PyPI. I also usually set the version number to `0.0.0` for applications when I'm using a tool that is a package build/publish tool as well as an application package manager.